storj/storj
# Make windows version more dynamic

Test:
- Have [goversioninfo](https://github.com/josephspurrier/goversioninfo) installed
- Tag a version, like v1.2.3: `git tag v1.2.3`
- Build a binary for windows: `make storagenode_windows_amd64`
- Observe the tag being the Product Version in the details tab:
![storagenode version](https://user-images.githubusercontent.com/812795/67416821-8877de00-f595-11e9-85da-dc8cecb57cf8.png)
